calendar-bundle/src/Picker/EventPickerProvider.php

```diff
@@ -45,7 +47,9 @@ public function supportsContext($context): bool
      */
     public function supportsValue(PickerConfig $config): bool
     {
-        return false !== strpos($config->getValue(), '{{event_url::');
+        $insertTagChunks = explode('%s', $this->getInsertTag($config, self::DEFAULT_INSERTTAG), 2);
```

Comment:
Can we encapsulate this in a `getInsertTagChunk()` method so we do not have to use multiple `explode()` calls?

Reply:
I'm not even sure if that should be named `insertTag` or if that should be something else? I mean, in fact it's just whatever string you want to provide and we're going to replace '%s' with the value. I just don't know if there's any other use case other than an insert tag 😄 wdyt?

Reply:
> I just don't know if there's any other use case other than an insert tag

I cannot think of one.

Reply:
Done 😄